msteams-platform/TOC.yml

```diff
@@ -105,7 +105,7 @@
       - name: Tabs
         href: ./resources/design/framework/tabs
       - name: Messaging extensions
-        href: ./resources/design/framework/compose-extensions
```

Comment:
We have another file with the same name as well, right? Did you change that too?

Reply:
That's what I just changed.